### PR TITLE
WINC-942: [bundle] Prep for 8.0.0 release

### PR DESCRIPTION
**Pull request by alinaryan**
This change bumps the operator and OCP version
according to the WMCO 8.0.0 release.

Ran 'make bundle' after changing the Makefile and 'config/' files.

### CODE REVIEW

File: bundle/manifests/windows-machine-config-operator.clusterserviceversion.yaml [hunk 1]
```diff
@@ -345,7 +345,9 @@ spec:
           - use
         serviceAccountName: windows-machine-config-operator
       deployments:
-      - name: windows-machine-config-operator
+      - label:
+          name: windows-machine-config-operator
```
Comment (jrvaldes) [inline bundle/manifests/windows-machine-config-operator.clusterserviceversion.yaml:349]: Is this change required?

**Review by jrvaldes — COMMENTED** [review]
@alinaryan  Thanks for working on this.

File: config/manifests/bases/windows-machine-config-operator.clusterserviceversion.yaml [hunk 2]
```diff
@@ -8,7 +8,7 @@ metadata:
     certified: "false"
     createdAt: REPLACE_DATE
     description: An operator that enables Windows container workloads on OCP
-    olm.skipRange: '>=6.0.0 <7.0.0'
+    olm.skipRange: '>7.0.0 <8.0.0'
```
Comment (mansikulkarni96) [inline config/manifests/bases/windows-machine-config-operator.clusterserviceversion.yaml:11]: should this be >=7.0.0?

File: bundle/manifests/windows-machine-config-operator.clusterserviceversion.yaml [hunk 3]
```diff
@@ -8,7 +8,7 @@ metadata:
     certified: "false"
     createdAt: REPLACE_DATE
     description: An operator that enables Windows container workloads on OCP
-    olm.skipRange: '>=6.0.0 <7.0.0'
+    olm.skipRange: '>7.0.0 <8.0.0'
```
Comment (mansikulkarni96) [inline bundle/manifests/windows-machine-config-operator.clusterserviceversion.yaml:11]: should this be >=7.0.0?

**Review by sebsoto — APPROVED** [review]

**Comment by mtnbikenc** [comment]
Priority PR has merged.
/hold cancel
